data/rulesets/deeds/scripts/world/objects/Weather.py

```diff
@@ -1,7 +1,7 @@
 # This file is distributed under the terms of the GNU General Public license.
 # Copyright (C) 1999 Al Riddoch (See the file COPYING for details).
 
-from random import *
+from random import randint
```

Comment:
This is unfortunately incorrect as "uniform" also is imported from "random".

Reply:
My bad. Fixed.